specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -21567,6 +21567,412 @@
+            "$ref": "./examples/ListWorkflows.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/workflows/{workflowName}": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Get workflow information by its ID for web site, or a deployment slot.",
+        "operationId": "WebApps_GetInstanceWorkflowSlot",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/siteNameParameter"
+          },
+          {
+            "name": "slot",
+            "in": "path",
+            "description": "Name of the deployment slot.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "workflowName",
+            "in": "path",
+            "description": "Workflow name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Workflow information returned.",
+            "schema": {
+              "$ref": "#/definitions/WorkflowEnvelope"
+            }
+          },
+          "404": {
+            "description": "Workflow with a name of {workflowName} does not exist.",
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GET a workflow": {
+            "$ref": "./examples/GetWorkflow.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/workflowsconfiguration/connections": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Get workflow app's configurations connections information by its ID for web site, or a deployment slot.",
+        "operationId": "WebApps_ListInstanceWorkflowsConfigurationConnectionsSlot",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/siteNameParameter"
+          },
+          {
+            "name": "slot",
+            "in": "path",
+            "description": "Name of the deployment slot.",
+            "required": true,
+            "type": "string"
```

Comment:
You have a lot of 'Name of the deployment slot' descriptions all through your json files. Assuming they are all logically exactly the same thing, I'd suggest refactoring them all to reference a common definition [or a few -- if they aren't strict dupes].

Reply:
Yep, I was following the same style defined in the rest of the definition. I can replace them in Logic Apps routes, but I'd want the other owners of other services in this definition to make changes to their routes.

Reply:
Updated the workflow definitions and a few other exact copies to have one universal slot reference. The other references have unique descriptions that I think do enhance the API documentation.